.github/workflows/kotlin-build.yml

```diff
@@ -0,0 +1,28 @@
+name: "Kotlin Build"
+
+on:
+  pull_request:
+    paths:
+      - "java/kotlin-extractor/**"
+      - "misc/bazel/**"
+      - "misc/codegen/**"
+      - "*.bazel*"
+      - .github/workflows/kotlin-build.yml
+    branches:
+      - main
+      - rc/*
+      - codeql-cli-*
+
+permissions:
+  contents: read
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - run: |
+          bazel query //java/kotlin-extractor/...
```

Comment:
This probably ignores `.bazelversion`, because iirc `bazel` is provided by the images team pre-installed. I don't know if they provide `bazelisk` too.

Reply:
Yes, bazelisk is provided (see https://github.com/actions/runner-images/blob/ubuntu22/20240407.1/images/ubuntu/Ubuntu2204-Readme.md#tools), and is actually available directly as  `bazel`, see https://github.com/github/codeql/actions/runs/8703530745/job/23869904067?pr=16117#step:3:8

Reply:
Nice! Then this is not a concern after all.